src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -593,3 +609,84 @@ def remove_image(self, pool_name, group_name, image_name, namespace=None):
             RbdService.validate_namespace(ioctx, namespace)
             ioctx.set_namespace(namespace)
             return group.remove_image(ioctx, image_name)
+
+
+@APIRouter('/block/pool/{pool_name}/group/{group_name}/snap', Scope.RBD_IMAGE)
+@APIDoc("RBD Group Snapshot Management API", "RbdGroupSnapshot")
+class RbdGroupSnapshot(RESTController):
+
+    RESOURCE_ID = "snapshot_name"
+
+    def __init__(self):
+        super().__init__()
+        self.rbd_inst = rbd.RBD()
+
+    @handle_rbd_error()
```

Comment:
I see in some endpoints you are using `@handle_rbd_error`, while in others `@RbdTask..` what's the reason for one or another?